addons/bestja_detailed_reports/views/detailed_reports.xml

```diff
@@ -0,0 +1,144 @@
+                                    ('user_can_moderate', '=', False),
+                            
+                           ]}"/>
+                            <button name="set_rejected" string="Odrzuć" type="object" attrs="{'invisible': [
+                                '|',
+                                    ('state', '=', 'rejected'),
+                                    ('user_can_moderate', '=', False),
+                            ]}"/>
+                        </div>
+                    </header>
+                    <div attrs="{'invisible': [('id', '=', False)]}">
+                    <separator string="Lista zebranych produktów" class="oe inline"/>
+                    <a href="/bestja_detailed_reports/static/grupy_towarowe.pdf">Pobierz opis grup towarowych</a> 
+                    </div>
+                    <div class="oe_edit_only oe_grey" attrs="{'invisible': [('id', '!=', False)]}">Wybierz projekt, do którego chcesz stworzyć raport szczegółowy.</div>
+                    <group attrs="{'invisible': [('id', '!=', False)]}">
+                        <field name="project" options="{'no_create': True}" attrs="{'readonly': [('id', '!=', False)]}"/>
+                    </group>
+                    <button string="Kontynuuj ▶" class="oe_edit_only" type="object" name="continue_action" attrs="{'invisible': [('id', '!=', False)]}"/>
+                    <div attrs="{'invisible': [('id', '=', False)]}">
+                        <div class="oe_title">
+                            <label for="state" class="oe_inline"/><field name="state" readonly="1" class="state oe_inline"/>
+                        </div>
+                        <br/><br/><br/>
+                        <div>
+                        Aby dodać grupy towarowe pojedynczo, naciśnij "dodaj grupę towarową" i wybierz ją z listy. Aby dodać wszystkie grupy, naciśnij "dodaj wszystkie grupy".<br/>
+                        Możesz zapisać raport, aby do niego wrócić. Po kliknięciu przycisku "Wyślij raport" nie ma możliwości zmiany wprowadzonych danych. Dlatego przed wysłaniem raportu należy 
+                        upewnić się, że wszystkie komórki zawierają prawidłowe dane.<br/>
+                        Klikając na przycisk "Pobierz opis grup towarowych" można dowiedzieć się jakie produkty należą do danej kategorii.
+                        </div>
+                        <field name="report_entries" options="{'no_create': True}">
+                            <tree editable="true" >
+                                <field name="commodity_code" readonly="1"/>
+                                <field name="commodity"/>
+                                <field name="tonnage"/>
+                                <field name="interested_organization" invisible="1"/>
+                            </tree>
+                        </field>
+                        <button string="dodaj wszystkie grupy" class="oe_link" type="object" name="add_all_commodity_groups" states="draft"/>
```

Comment:
Add `oe_edit_only` class, remove `states="draft"` (since if someone can edit the report she should be also to also use this button).